static/js/components/UserInfoCard_test.js

```diff
@@ -0,0 +1,31 @@
+/* global SETTINGS: false */
+import React from 'react';
+import { shallow } from 'enzyme';
+import { assert } from 'chai';
+
+import { USER_PROFILE_RESPONSE } from '../constants';
+import R from 'ramda';
+import UserInfoCard from './UserInfoCard';
+
+describe('UserInfoCard', () => {
+  let clone;
+
+  const renderInfoCard = profile => (
+    shallow(<UserInfoCard profile={profile} />)
+  );
+
+  beforeEach(() => {
+    clone = R.clone(USER_PROFILE_RESPONSE);
+  });
+
+  it('should show email if its not his own card', () => {
+    SETTINGS.user.username = "My user";
+    let chip = renderInfoCard(clone);
+    assert.equal(chip.find(".profile-email").text(), clone.email);
+  });
+
+  it('should not show email if its personal card', () => {
+    let chip = renderInfoCard(clone);
+    assert(chip.find(".profile-email").isEmpty(), 'email is shown');
+  });
+});
```

Comment:
Could you re-write the test descriptions to be gender-neutral and to use the user-based language we usually use? Something like `'should show email if the user is viewing their own profile'`, etc.